content/kubermatic/main/tutorials-howtos/metering/_index.en.md

```diff
@@ -167,9 +167,9 @@ Kubelet Metrics:
 - machine_cpu_cores
 - machine_memory_bytes
 
-Machine controller Metrics:
+Other Metrics:
```

Comment:
I don't think this should be called "other metrics", since it's metrics coming from kube-state-metrics. Can we specify that somewhere? It seems that kube-state-metrics is now a requirement for metering, but the page doesn't even mention it as pre-requisite (it's an optional addon not installed into all clusters).

Reply:
yeah I changed the titel. Turns our kube-state-metrics is deployed by our monitoring controller by default:
https://github.com/kubermatic/kubermatic/blob/main/pkg/controller/seed-controller-manager/monitoring/resources.go#L89